docs/dashboard.md

```diff
@@ -15,42 +15,65 @@ across all your clusters. You can also invite users to your workspace.
    <a href="https://zora-dashboard.undistro.io/" class="md-button">Try Zora Dashboard</a>
 </div>
 
-Currently, it's free for up to 3 clusters.
-Please [contact us](https://undistro.io/contact){:target="_blank"} if you need unlock more clusters in Zora Dashboard.
+Zora Dashboard offers a starter plan for 14 days, after which it will revert to the free plan which provides access for 2 clusters with up to 10 nodes per cluster.
```

Comment:
It would be interesting to give highlight  " ... __starter plan__ for __14 days__

Reply:
I've made it bold and italic, check it out and see what you think